src/components/AccountCreation/AccountCreation.tsx

```diff
@@ -24,46 +35,66 @@ const AccountCreation = ({
   };
 
   return (
-    <form onSubmit={handleCreateAccount}>
-      <div>
-        <h2>Create Account</h2>
-        <div>
-          <label>
-            Name:
-            <input
-              required
-              type="text"
-              value={name}
-              onChange={(e) => setName(e.target.value)}
-            />
-          </label>
-        </div>
-        <div>
-          <label>
-            Email:
-            <input
-              required
-              type="text"
-              value={email}
-              onChange={(e) => setEmail(e.target.value)}
-            />
-          </label>
-        </div>
-        <div>
-          <label>
-            Password:
-            <input
-              required
-              type="password"
-              value={password}
-              minLength={8}
-              onChange={(e) => setPassword(e.target.value)}
-            />
-          </label>
-        </div>
-        <button type="submit">Create Account</button>
-      </div>
-    </form>
+    <ThemeProvider theme={theme}>
+      <Container>
+        <Box
+          display="flex"
+          justifyContent="center"
+          alignItems="center"
+          minHeight="100vh"
+        >
+          <form onSubmit={handleCreateAccount}>
+            <Typography
+              gutterBottom
+              component="h3"
+              variant="h3"
+              fontFamily="Pacifico"
+            >
+              Create Account
+            </Typography>
+            <div>
+              <TextField
+                required
+                fullWidth
+                id="name-field"
+                type="text"
+                label="Name"
+                margin="normal"
+                defaultValue={name}
+                onChange={(e) => setName(e.target.value)}
+              />
+            </div>
+            <div>
+              <TextField
+                required
+                fullWidth
+                id="email-field"
+                label="Email"
+                type="email"
+                margin="normal"
+                defaultValue={email}
+                onChange={(e) => setEmail(e.target.value)}
+              />
+            </div>
+            <div>
+              <TextField
+                required
+                fullWidth
+                id="password-field"
+                label="Password"
+                minLength={8}
```

Comment:
@anniethewebcoder It looks like `minLength` doesn't work on these components unless you pass it as as part of `inputProps`: 

https://mui.com/material-ui/api/text-field/#advanced-configuration

```suggestion
                inputProps={{
                  minLength: 8,
                }}
```

Reply:
Ah, I didn't even test that part. Thanks! I changed it.